clusterloader2/pkg/measurement/common/wait_for_controlled_pods.go

```diff
@@ -78,6 +78,7 @@ type waitForControlledPodsRunningMeasurement struct {
 	kind                  string
 	selector              *measurementutil.ObjectSelector
 	operationTimeout      time.Duration
+	countErrorMargin      int
```

Comment:
I know we haven't been doing it so far, but can we start documenting the fields here? They are, in a way, forming an external API (measurement api exposed in the yaml files) and deserve a sentence of documentation if meaning is not 100% clear.

Reply:
Done

Reply:
Documentation in golang should be "a complete sentence that begins with the name of the element it describes" - https://blog.golang.org/godoc